MapboxCoreNavigation/Constants.swift

```diff
@@ -170,3 +170,18 @@ extension Notification.Name {
      */
     public static let routeControllerDidPassVisualInstructionPoint = MBRouteControllerDidPassVisualInstructionPoint
 }
+
+/**
+ The minimum distance between location updates before a location update is filtered out.
+ */
+public var RouteControllerMinimumDistanceForProgressNotification: CLLocationDistance = 1
+
+/**
+ The minimum difference between the user's course before a location update is filtered out.
+ */
+public var RouteControllerMinimumDifferenceBetweenCourse: CLLocationDirection = 1
+
+/**
+ The minimum elapsed time between location updates before a location update is filtered out.
+ */
+public var RouteControllerMinimumElapsedTimeBetweenLocations: TimeInterval = 60
```

Comment:
Do we actually want to throttle location updates to once a second? If we connect to an external GPS device or car, wouldn’t we want to get the more frequent location updates? Or do we think such updates will rarely also look identical in terms of location and course?

Reply:
So we would only filter when all three cases are met:

* The new location is within 1 meter of the previous location.
* The new course is within 1 degree of the previous course.
* The new location is less than 60 seconds old.

Reply:
I think if location updates are coming from a high frequency device, and they are very close in terms of proximity, this is even more of a reason to throttle progress events.